README.md

```diff
@@ -1,198 +1,163 @@
-# Eik Node Client
+# @eik/node-client
 
-The Eik Node.js client facilitates switching between local and production assets in Node.js apps based on values
-provided by an `assets.json` metafile.
+The Eik Node.js client facilitates loading Eik config and providing URLs to assets on an Eik server or in local development plus loading import maps from the Eik server.
 
 ## Install
 
 ```sh
-npm install @eik/client
+npm install @eik/node-client
 ```
 
 ## Basic usage
 
-### include the client in your node apps
-
 ```js
-const Client = require('@eik/client');
-```
+import EikNodeClient from '@eik/node-client';
```

Comment:
Is the word `Node` of any value here?

Reply:
Yes. This is very node specific. We might make the same feature for java, go, deno etc. so having a convention on prefixing these is handy.